examples/ex1.cpp

```diff
@@ -63,6 +69,10 @@ int main(int argc, char *argv[])
                   " isoparametric space.");
    args.AddOption(&static_cond, "-sc", "--static-condensation", "-no-sc",
                   "--no-static-condensation", "Enable static condensation.");
+   args.AddOption(&pa, "-p", "--pa", "-no-p", "--no-pa",
+                  "Enable Partial Assembly.");
+   args.AddOption(&device, "-d", "--device",
+                  "Device configuration, e.g. 'cuda', 'omp', 'raja', 'occa'.");
    args.AddOption(&visualization, "-vis", "--visualization", "-no-vis",
                   "--no-visualization",
                   "Enable or disable GLVis visualization.");
```

Comment:
We should probably check for incompatible options here like static condensation  + partial assembly, or anything specific device configurations don't work with.

Reply:
I think it should fail inside the library, the example (i.e., user code) can't check all combinations.

Reply:
There is a warning about this here:

https://github.com/mfem/mfem/blob/okina/fem/bilinearform.cpp#L133-L138